RELEASING.md

```diff
@@ -0,0 +1,39 @@
+# Scala.js Release Process
+
+1. Clean up bugs:
+    - Make sure all bugs assigned to the milestone are closed.
+    - Make sure all fixed bugs are assigned to the milestone ([this query][1]
+      should not return anything).
+1. Create a "Version x.y.z." commit ([example][2]) and push it to a branch on
+   your fork.
+1. Ping people on the commit for review.
+1. Once you have LGTM, push to master (do *not* create a merge commit).
+1. Testing (post results as comments to commit):
+    - Nightly
+    - Weekly
+    - [Manual testing][3]
+1. If all tests pass, tag the commit with the release version.
+1. Perform [manual testing][3] that needs the tagging (source maps).
+1. Publish:
+    - Sonatype, bintray (`./script/publish.sh`)
+    - CLI to website (simple `scp`)
+    - Docs to website: Use
+      `~/fetchapis.sh <full sjs version> <binary sjs version>` on the webserver
+      once artifacts are on maven central.
+1. Once artifacts are on maven central, create a "Towards x.y.z." commit
+   ([example][4]). Same procedure for push to master than the "Version x.y.z."
+   commit.
```

Comment:
The procedure is not the same here. We create a DO NOT MERGE PR for the CI to kick in. But then we fast-forward the commit without clicking the Merge button. Example: #2473.

Afterwards, the PR appears as Merged anyway, because all its commits are in master, but that's not the result of clicking the Merge button. That's why I tend to rename the PR to remove the DO NOT MERGE prefix afterwards, otherwise it's very surprising and confusing in the list of closed PRs.